tmp.yaml

```diff
@@ -0,0 +1,61 @@
+  tikv:
```

Comment:
Do we need this file in the docs repo?  @DanielZhangQD

Reply:
NO. @july2993 Please remove this file.

Reply:
remove in https://github.com/pingcap/docs-tidb-operator/pull/1167/commits/bcb5d11f75b0b16b7c491a1f81c3b28a446e9fa4
sorry, it's by mistake